test/TPP/tpp/tpp-invalid.mlir

```diff
@@ -21,7 +21,7 @@ func.func @tpp_add_invalid(%arg0: f32, %arg1: f32) {
 
 func.func @tpp_relu_invalid(%arg0: memref<1x2xf32>, %arg1: memref<2x1xf32>) -> memref<2x1xf32> {
 
-  // expected-error @below {{'tpp.relu' op requires all operands to have the same type}}
```

Comment:
I think here you slightly change what the test was trying to do: check that input and output have the same operands. Since we have a single argument I think we can remove it.